src/main.cpp

```diff
@@ -170,9 +186,22 @@ int main(int argc, char *const argv[]) {
 
     P4PRUNER::PrunerConfig pruner_conf;
 
+    switch (tolower(options.bug_type)) {
+    case 'c':
+        options.validation_bin = nullptr;
+        INFO("Ignoring validation bin for crash bug");
+        break;
+    case 'v':
+        break;
+    default:
+        ::error("Please enter a valid bug type. V for validation or C for "
+                "crash bug");
+        exit(EXIT_FAILURE);
+    }
+
     if (options.config_file) {
-        pruner_conf = get_config_from_json(options.config_file,
```

Comment:
Considering we now know the exact bug type, this if-then-else casing also needs to be changed.